samples/app-anonymous-users/csharp/ChatHub.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <copyright file="ChatHub.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+using Microsoft.AspNetCore.SignalR;
+using System.Threading.Tasks;
+
+namespace AnonymousUsers
+{
+    /// <summary>
+    /// To make calls to specific clients, use the properties of the Clients object.
+    /// SendMessage sends a message to all connected clients, using Clients.All.
+    /// </summary>
+    public class ChatHub : Hub
+    {
+        public async Task SendMessage(string description, int count)
```

Comment:
/// This method is used to sends a message to all connected clients using Clients.All and it will sync the data in UI.

Reply:
Fixed